modules/fedimint-gw-client/src/pay.rs

```diff
@@ -0,0 +1,17 @@
+/// State machine that pays a lightning invoice on behalf of a federation user.
```

Comment:
Module docs begin with `//! `, `/// ` only works for docs attached to things defined right below (and module comments kinda relate to the whole file instead).

But if you do that the `#[aquamarine]` attribute macro doesn't work. So either you could move the comment to `lib.rs` as a normal `///` doc comment above the `mod` decalaration or you attach them to some struct defined inside the module.

Reply:
Added some dummy structs so the diagrams appear